src/main/java/io/fabric8/maven/docker/config/ImageConfiguration.java

```diff
@@ -153,6 +154,21 @@ public String initAndValidate(ConfigHelper.NameFormatter nameFormatter, Logger l
         return minimalApiVersion;
     }
 
+    public static ImageConfiguration getDefaultImageConfiguration(MavenProject project, String defaultImageName) {
```

Comment:
Can we keep the dependency on Maven low ? actually I would keep the extraction of the default name from maven coordinates in the Mojos.

The reason is, that eventually, we might want to separate the Maven and non-Maven parts (see https://github.com/fabric8io/fabric8-build for more details of these plans).